public_html/install/abantecart_database.sql

```diff
@@ -1670,8 +1670,8 @@ INSERT INTO `ac_settings` (`group`, `key`, `value`) VALUES
 -- Appearance
 
 ('appearance','storefront_width','100%'),
-('appearance','config_logo','image/18/73/3.png'),
-('appearance','config_mail_logo','image/18/73/3.png'),
+('appearance','config_logo_1','image/18/73/3.png'),
```

Comment:
I do not like that. Default should stay as it was before. 
1. People will loose logos on upgrade. 
2. It is not clear and guarantied that default will be 1

Reply:
upgrade script will write later. It's a installation sql.

Reply:
It has to stay as it was with default values.